.circleci/config.yml

```diff
@@ -125,20 +169,50 @@ jobs:
     steps:
       - assume-role-and-persist-workspace:
           aws-account: $AWS_ACCOUNT_PRODUCTION
-  terraform-init-and-apply-to-development:
+  terraform-init-and-plan-development:

```

Comment:
Can we add the plan and terraform compliance steps as part of the checks that run automatically for every commit so PR reviewers can see the result of the terraform compliance checks before approving?

Reply:
Sure, will update the workflow